### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by gagath** [forrest-marshall/serde-hex#13]
Dear maintainer,

A set of pull requests have been waiting for integration since some months without any feedback. Maintaining software is not an easy task, and the motivation may go away once there is no more interest in the project.

If you are willing to not support it anymore, could you provide the community interested in maintaining it a way to reuse this repository and the cargo crate, or should we consider creating a fork and finding a new name for crates.io? At least for this last option we can do on our own, but answering this request would help the process for sure.

Thanks in advance for your consideration.

**Comment by cryptoquick** [comment]
It's been nearly two years since this has been asked of @fspmarshall , I don't suppose you're maintaining a fork, @MicroJoe ?

**Comment by gagath** [comment]
@cryptoquick No, I am not maintaining a fork currently as my project which was using it is on hold. Feel free to do so, if you can merge the pending pull requests that would be great!
